vagrant/provisioning/aws-zookeeper.yml

```diff
@@ -0,0 +1,41 @@
+---
+- hosts: localhost
+  roles:
+    - pki-aws-client
+    - zookeeper
+
+  vars_prompt:
+    - name: root_folder
```

Comment:
I'm thinking to include these in the other playbooks too, for better documentation of the simple facts.  Some of the LDAP and Drools facts are a little too complex for this approach though.